gym/envs/registration.py

```diff
@@ -400,9 +409,7 @@ def spec(self, path: str) -> EnvSpec:
             versions = self._versions(namespace, name)
             version_not_found_error_msg += ", ".join(
                 map(
-                    lambda version: env_id_from_parts(
-                        getitem(version, 1), name, getitem(version, 0)  # type: ignore
-                    ),
+                    lambda version: env_id_from_parts(version[1], name, version[0]),
```

Comment:
It would be even nicer if we returned a `namedtuple` from `_versions` so we could do (`id.namespace, name, id.version`) instead of indexing into the tuple. This isn't necessary but just something I thought of.

Reply:
I like this idea!